core/server/src/main/java/tachyon/web/WebInterfaceDownloadServlet.java

```diff
@@ -69,7 +70,8 @@ public WebInterfaceDownloadServlet(FileSystemMaster fsMaster) {
   @Override
   protected void doGet(HttpServletRequest request, HttpServletResponse response)
       throws ServletException, IOException {
-    if (PlainSaslServer.AuthorizedClientUser.get() == null) {
+    if (SecurityUtils.isSecurityEnabled(MasterContext.getConf())
```

Comment:
Sounds good. Added checking authentication enabled in the method.

For the special handling here, we still have to check if security (authentication + authorization) is enabled explicitly, so as to not impact non-security logic if using default conf.